src/throttler.guard.ts

```diff
@@ -18,49 +23,37 @@ export class ThrottlerGuard implements CanActivate {
     private readonly reflector: Reflector,
   ) {}
 
-  // TODO: Return true if current route is in excludeRoutes.
   async canActivate(context: ExecutionContext): Promise<boolean> {
     const handler = context.getHandler();
+    const classRef = context.getClass();
     const headerPrefix = 'X-RateLimit';
 
     // Return early when we have no limit or ttl data.
     const routeOrClassLimit = this.reflector.getAllAndOverride<number>(THROTTLER_LIMIT, [
       handler,
-      context.getClass(),
+      classRef,
     ]);
     const routeOrClassTtl = this.reflector.getAllAndOverride<number>(THROTTLER_TTL, [
       handler,
-      context.getClass(),
+      classRef,
     ]);
-    // check if specific limits are set at class or route level
-    // use global options if not
+
+    // Check if specific limits are set at class or route level, otherwise use global options.
     const limit = routeOrClassLimit || this.options.limit;
     const ttl = routeOrClassTtl || this.options.ttl;
     if (typeof limit === 'undefined' || typeof ttl === 'undefined') {
       return true;
```

Comment:
Just a callout, as we're going to be requiring options from the `forRoot` of the module registration, I don't think we need to worry about this `if limi || ttl` check. This should never really be undefined.